Makefile.am

```diff
@@ -102,6 +103,11 @@ ppmclibs/Makefile: ppmclibs/Makefile.PL
 $(PERL_MODULE): ppmclibs/Makefile
 	$(MAKE) -C ppmclibs
 
+$(OPENVSWITCH_SERVICE):
+	rm -f $@ $@.tmp
+	sed -e 's,@pkglibexecdir[@],$(pkglibexecdir),g' '$(top_srcdir)/$@.in' > '$(top_srcdir)/$@.tmp'
```

Comment:
It doesn't seem appropriate here. The file is not usable as-is, so it should be marked as such by calling it `.in` and not editing it in-place. It's not like the other cases, where the file is usable as-is when os-autoinst is run out of the git checkout. If anything we could drop the use of the `.tmp` file I guess, it's not doing a whole lot, I just copied it out of some autotools docs I think.